types/V1.mli

```diff
@@ -522,6 +522,27 @@ module type IPV6 = sig
   include IP
 end
 
+(** {1 ICMP module} *)
+module type ICMP = sig
+  include DEVICE
+
+  type ipaddr
+  type buffer
+
+  val pp_of_error : Format.formatter -> error -> unit
```

Comment:
I think the normal convention is to call this `pp_error`. It prints as a side-effect, rather than converting from one thing to another.